lib/process-services-cloud/src/lib/app/components/app-details-cloud/app-details-cloud.component.ts

```diff
@@ -22,9 +22,10 @@ import { TranslatePipe } from '@ngx-translate/core';
 import { MatIconModule } from '@angular/material/icon';
 import { MatCardModule } from '@angular/material/card';
 
+/** @deprecated this component will be removed because it's unused */
```

Comment:
Please apply that comment to all places - please add jira number to each deprecated comment to track those places